src/VisualStudio/Core/Def/SymbolSearch/VisualStudioSymbolSearchService.cs

```diff
@@ -53,6 +55,7 @@ internal partial class VisualStudioSymbolSearchService : AbstractDelayStartedSer
             _localSettingsDirectory = new ShellSettingsManager(serviceProvider).GetApplicationDataFolder(ApplicationDataFolder.LocalSettings);
```

Comment:
This looks like a pre-existing case of thread affinity in this MEF part's activation path. Please plan on fixing this and any other offenders when you can unless you're absolutely sure that all possible import paths always are and will be on the UI thread already -- in which case, it would be great to see `ThreadHelper.ThrowIfNotOnUIThread()` or equivalent at the start of each of these importing constructors.